.github/CODEOWNERS

```diff
@@ -1 +1,27 @@
-*   @abbashus @adnapibar @anasalkouz @andrross @Bukhtawar @CEHENKLE @dblock @dbwiddis @dreamer-89 @gbbafna @kartg @kotwanikunal @mch2 @msfroh @nknize @owaiskazi19 @peternied @reta @Rishikesh1159 @ryanbogan @sachinpkale @saratvemulapalli @setiah @shwetathareja @sohami @tlfeng @VachaShah
+# CODEOWNERS manages notifications, not PR approvals
+# For PR approvals see /.github/workflows/maintainer-approval.yml 
+
+# Files have a single rule applied, the last match decides the owner
+# If you would like to more specifically apply ownership, include existing owner in new sub fields
+
+# To verify changes of CODEOWNERS file
+# In VSCode
+#   1. Install extension https://marketplace.visualstudio.com/items?itemName=jasonnutter.vscode-codeowners
+#   2. Go to a file
+#   3. Use the command palette to run the CODEOWNERS: Show owners of current file command, which will display all code owners for the current file.
+
+# Default ownership for all repo files
+* @abbashus @adnapibar @anasalkouz @andrross @Bukhtawar @CEHENKLE @dblock @dbwiddis @dreamer-89 @gbbafna @kartg @kotwanikunal @mch2 @msfroh @nknize @owaiskazi19 @reta @Rishikesh1159 @ryanbogan @sachinpkale @saratvemulapalli @setiah @shwetathareja @sohami @tlfeng @VachaShah
+
+/modules/transport-netty4/ @peternied
+
+/plugins/identity-shiro/ @peternied
+
+/server/src/main/java/org/opensearch/extensions/ @peternied
+/server/src/main/java/org/opensearch/identity/ @peternied
+/server/src/main/java/org/opensearch/threadpool/ @peternied
+/server/src/main/java/org/opensearch/transport/ @peternied
+
+/.github/ @peternied
```

Comment:
Just to confirm -- changes to these specific areas will *also* notify the default folks, right?

Reply:
Unfortunately no, only one line is applied per file in the PR.  If you want to be notified about an area you'd need to add yourself to these areas sections.

Note; with the CHANGLOG on the root of the project is a requirement for the vast majority of changes so that will pull in the 'default' groups.

Reply:
Ahh... got it.

I think that's still good. I just wanted to make sure that people wouldn't suddenly stop receiving notifications for some changes (i.e. "@peternied is the **only** one who gets notified when changes go in for extensions, identity, threadpool, and transport!").

As you said, since those changes are all going to be hitting CHANGELOG.md too, we'll all still get notified anyway.

My approval still stands :+1: